Strings/Cards.jsonc

```
@@ -131,41 +131,41 @@
 	"リストにないカラーにしたい場合は iOS 14 以降へアップデートしてください。": {
 		"ja": "リストにないカラーにしたい場合は iOS 14 以降へアップデートしてください。",
 		"en": "Please update to iOS 14 or later to set custom colors.",
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"zh-Hans": "请更新到 iOS 14 或更高版本以设定自定义颜色。",
+		"zh-Hant": "請升級至 iOS 14 或以上版本以設定自訂顏色。",
+		"zh-HK": "請更新至 iOS 14 或以上版本以設定自訂顏色。"
 	},
 
 	"画像と読み取り日時": {
 		"ja": "画像と読み取り日時",
 		"en": "Card Image & Scanned Date",
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"zh-Hans": "卡片图像及读取时间",
+		"zh-Hant": "卡片圖像及讀取日期",
+		"zh-HK": "卡片圖像及讀取日期"
 	},
 
 	"画像のみ": {
 		"ja": "画像のみ",
 		"en": "Card Image",
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"zh-Hans": "卡片图像",
+		"zh-Hant": "卡片圖像",
+		"zh-HK": "卡片圖像"
 	},
 
 	"未設定": {
 		"ja": "未設定",
-		"en": "Not Set",
+		"en": "Not Configured",
 		"zh-Hans": "没有设定",
 		"zh-Hant": "沒有設定",
 		"zh-HK": "未設定"
 	},
 
 	"名称": {
 		"ja": "名称",
-		"en": "Display Name",
+		"en": "Name",
```

I'm sorry. I want to emphasize that this is a "Display Name", so please don't change it.
```suggestion
		"en": "Display Name",
```